VideoWeb/VideoWeb/Extensions/ConfigureServicesExtensions.cs

```diff
@@ -146,21 +146,26 @@ public static IServiceCollection AddCustomTypes(this IServiceCollection services
             };
 
             var connectionStrings = container.GetService<ConnectionStrings>();
-            services.AddSignalR()
-                .AddAzureSignalR(options =>
-                {
-                    options.ConnectionString = connectionStrings.SignalR;
-                    options.ClaimsProvider = context => context.User.Claims;
-                })
-                .AddNewtonsoftJsonProtocol(options =>
-                {
-                    options.PayloadSerializerSettings.Formatting = Formatting.None;
-                    options.PayloadSerializerSettings.ContractResolver = contractResolver;
-                    options.PayloadSerializerSettings.DateTimeZoneHandling = DateTimeZoneHandling.Utc;
-                    options.PayloadSerializerSettings.Converters.Add(
-                        new StringEnumConverter());
-                })
-                .AddHubOptions<EventHub.Hub.EventHub>(options => { options.EnableDetailedErrors = true; });
+            services.AddSignalR(hubOptions =>
+            {
+                hubOptions.EnableDetailedErrors = true;
```

Comment:
Is this duplicated by line 168? And should we instead put the new options into that AddHubOptions call?

Reply:
good catch, I totally missed this, I will move the extra parameters there